README.md

```diff
@@ -20,6 +20,11 @@ colcon build
 source install/setup.sh
 ```
 
+Test the repository (optional)
+```
+colcon test --event-handlers console_cohesion+ --return-code-on-test-failure --packages-select create3_control
```

Comment:
e' sufficiente fare `colcon test`, il resto non serve

Reply:
Ho visto dalla CI che viene lanciato con quelle opzioni colcon test. Usandolo così si ottiene visivamente lo stesso output della CI anche in locale. Quindi era sia per consistenza che per comodità (in quanto l'output viene chiaramente visualizzato da terminale). Detto ciò, può benissimo essere che ci siano delle opzioni che in locale non siano necessarie. Controllerò.

Reply:
Nessuna delle opzioni è necessaria in locale, anzi sono proprio non raccomandate

Reply:
La prima opzione stampa l output dei test in una botta sola quando il test è finito. Questo non va bene per due motivi: inondare il terminale di roba è male, anche perché potrebbe benissimo non starci tutto quindi hai un log a metà. Poi se sei in locale puoi benissimo controllare l output del test che è fallito dai log salvati su disco. Questo si fa nella ci perché non hai accesso a quei log e per salvare banda nel caricare le cose su github.
La seconda opzione ritorna un error code dal comando. È inutile se stai chiamando colcon test a mano e non all interno di uno script.
La terza opzione seleziona quali pacchetti testare, tu lo vuoi testare tutti.

Non forzare mai opzioni non necessarie sugli utenti di un pacchetto open source.
Il tuo readme attuale dice: per fare i test devi fare così. Ma in realtà dovrebbe dire: i test non hanno nulla di particulare eseguili come vuoi, e colcon test è la base, poi le opzioni le metti come vuoi o come ti serve